### DISCUSSION TITLE
How to stop plotting?

### DISCUSSION THREAD

**Discussion by jimmy6** [Chia-Network/chia-blockchain#3097] [Q&A] [answered]
I created 10 parallel pot and size 208Gib. But my Harddisk space is just 600G. So endup my remaining space is 0. How to stop plotting?

The log showing
Only wrote 0 of 283785 bytes at offset 23154155800 to "C:\\chia\\temp\\plot-k33-2021-04-29-18-11-c14469622eec573d2dc46bab9c8dc56e230adca483f0cf86b452d299bbaedd3d.plot.table1.tmp"with length 23154157352. Error 1. Retrying in five minutes.

**Comment by bnkbnkj** [comment, marked as answer]
Close Chia program. Will stop all plotting. Open back up and it's reset to 0 plotting.

**Reply by nomadengineer** [reply]
Actually the plotting keep on going in the background and to stop it you need to kill the chia process!

**Comment by zjshen14** [comment]
Go to the task manager to kill the `chia` process, and delete *.tmp files to release space

**Reply by im2c0ol** [reply]
Don't do this, it will locked up your full node anyway. You'll have to check the process through PowerShell then kill one of the chia process and monitor which plot's temp file stop writing and delete all those temp files from that plot to clear up space. I done it once and works, but didn't remember what was the PowerShell script.

**Reply by deelkar** [reply]
you can use resource monitor in drive tab to find that PID, and you can show PIDs in Taskmanager, this makes terminating the right process quite simple
